packages/web-config-server/src/apiV1/dataBuilders/generic/count/countCalculatedValuesPerOrgUnit.js

```diff
@@ -0,0 +1,52 @@
+import groupBy from 'lodash.groupby';
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import {
+  calculateOperationForAnalytics,
+  getDataElementsFromCalculateOperationConfig,
+} from '/apiV1/dataBuilders/helpers';
+
+class CountCalculatedValuesPerOrgUnit extends DataBuilder {
+  async build() {
+    const { operation, dataClasses } = this.config;
+    const dataElementCodes = getDataElementsFromCalculateOperationConfig(operation);
+    const { results } = await this.fetchAnalytics(dataElementCodes);
+    const analyticsByOrgUnit = groupBy(results, 'organisationUnit');
+    const calculatedValues = Object.values(analyticsByOrgUnit).map(analytics =>
+      calculateOperationForAnalytics(analytics, operation),
+    );
+
+    const data = Object.entries(dataClasses).map(([dataClass, condition]) => {
+      let count = 0;
+      calculatedValues.forEach(value => {
+        if (checkValueSatisfiesCondition(value, condition)) {
+          count++;
+        }
+      });
+
+      return {
+        name: dataClass,
+        value: count,
+      };
+    });
+
+    return { data };
+  }
+}
+
+export const countCalculatedValuesPerOrgUnit = async (
+  { dataBuilderConfig, query, entity },
```

Comment:
@edmofro, Is this a model conflict?

Reply:
Yep, thanks @Alastair-L! Added to my list.

Reply:
Resolved in https://github.com/beyondessential/tupaia/pull/1484/commits/128b6edfc558a68c621c983061d0f46b300a19c2